### PR TITLE
Feature: Added full-width background to Stepper

### PR DESCRIPTION
**Pull request by robinpiets**
Please squash and merge

---

## Result (update):
image

## In the design there is more spacing at the bottom than the top:
image

### CODE REVIEW

**Comment by svenjens** [comment]
Is the margin correct? On the bottom seems to be much more space

**Review by svenjens — COMMENTED** [review]
There seems to be more space at the bottom than on the top

File: apps/client/src/routes.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ export const routeConfig = [
...
     ),
   },
+  {
+    exact: true,
+    name: "Stepper",
+    path: "/:slug/Stepper",
+    component: React.lazy(() =>
+      import(/* webpackPrefetch: true */ `./pages/StepperPage`)
+    ),
+  },
```
Comment (afjlambert) [inline apps/client/src/routes.js]: Did you add this intentionally?

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/routes.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/routes.js]: Nope. By accident. Removing it now

**Comment by robinpiets** [comment]
> There seems to be more space at the bottom than on the top

The spacing had to do with the Nav component. You are free to place as much space as required.

This is the default:

image

**Review by svenjens — APPROVED** [review]

**Pull request merged**